uui-components/src/navigation/MainMenu/MainMenu.tsx

```diff
@@ -232,6 +232,11 @@ export class MainMenu extends React.Component<MainMenuProps, MainMenuState> {
                     };
 
                     let menuItems = [appLogoItem, ...childrenItems];
+
+                    if (!this.props.appLogoUrl) {
```

Comment:
just add  ogo via unshift only if it passed to the props, like it did for customerLogoUrl